src/azul/plugins/__init__.py

```diff
@@ -191,3 +192,10 @@ def drs_uri(self, drs_path: str) -> str:
         This method is typically called by the service.
         """
         raise NotImplementedError
+
+    @abstractmethod
+    def file_url(self, file_uuid: str, file_version: str) -> Optional[str]:
+        """
+        HCA v1 file URL, if available.
```

Comment:
```suggestion
        A URL pointing at the specified (ot latest) version of the specified file in the underlying repository.
```

We want this to be an abstraction.